src/connectedk8s/HISTORY.rst

```diff
@@ -2,6 +2,9 @@
 
 Release History
 ===============
+1.5.3
+++++++
+* Add support to get AKS-EE kubernetes distro
```

Comment:
It is suggested to specify the related command of this change.
```suggestion
* `az xxx`: Add support to get AKS-EE kubernetes distro
```

Reply:
There is no command related to this change. This change introduces support to a new infrastructure and distribution. Basically, whenever the regular onboarding command `az connectedk8s connect` command is run on aks_edge_essential infrastructure (node), this change will return the appropriate name of the distribution for aks-edge-essential node.